### ISSUE TITLE
@react-three/drei contains broken CDN link in v9.58.5

### ISSUE THREAD

**Issue by Thenlie** [adrianhajdin/project_threejs_ai#29]
The problem here is that https://github.com/pmndrs/drei references a broken CDN link in their old package versions. So you will need to do a couple of things to fix this:

1. Update `@react-three/drei`
```sh
npm install @react-three/drei --latest
```
2. Remove [this line](https://github.com/adrianhajdin/project_threejs_ai/blob/main/client/src/canvas/Shirt.jsx#L44) as that prop is no longer valid on `<Decal />`
> **Note** See https://github.com/pmndrs/drei/blob/master/src/core/Decal.tsx#L7 for a list of props

---

Hope this helps! 👍 

I am happy to create a PR with these changes, but it does not look like this is actively maintained. So leaving this here in case others come across the same issue. 

## Error:

![image](https://github.com/adrianhajdin/project_threejs_ai/assets/41388783/c6bc9f3e-1445-4bbb-9c4e-015d3dabce8d)

**Comment by K4R7IK** [comment]
Thanks for letting us know. Right now, I was struggling to figure out why it wasn't working.

**Comment by naman2703** [comment]
I have the exact same issue brother. Did you find a fix to the issues then???

**Comment by Thenlie** [comment]
> I have the exact same issue brother. Did you find a fix to the issues then???

@naman2703 Yep, the entire solution is posted here.

**Comment by Thenlie** [comment]
> just need to remove the specific line 46 only? because it's still not working after removing the line 46. showing the same errors again.

You also need to update the package.

**Comment by naman2703** [comment]
> > just need to remove the specific line 46 only? because it's still not working after removing the line 46. showing the same errors again.
> 
> You also need to update the package.

Yes I updated that package @react-three/drei as well. But it still doesn't work. Same errors

**Comment by naman2703** [comment]
can you like help me out?? need this project up and working. It used to work back then but now it just doesn't. whenever I run it, it display the website for a second and then vanishes.

**Comment by K4R7IK** [comment]
@naman2703 Ping me up. I'll help you out. Do you use discord? I'll drop mine, so ping me there. 
**Discord: 402373075905282069**

**Comment by naman2703** [comment]
can't find you on discord. ping me instead - username - theguy2703

**Comment by jashan1926** [comment]
just use custom .hdr file like this `<Environment files={"little_paris_under_tower_1k.hdr"} />`  you can download the file from [here](https://polyhaven.com/a/little_paris_under_tower)  use hdr and 1k setting to download file and put the file in public folder.

**Comment by naman2703** [comment]
I am a noob, didn't really get what you are tryna say here. Where do I put the code of this file?

**Comment by naman2703** [comment]
Oh god, it magically started working. How did it work tho? how did this random file make the website work?

**Comment by naman2703** [comment]
It magically started working damn. How did a random file make it work? But the AI doesn't generate anything now. As soon as I give it a request the website breaks and goes all white. what may be the issue?

**Comment by jashan1926** [comment]
If this AI was working before you added that environment, then the issue is likely not related to `<Environment />`. Instead, it is most likely an API key issue. If you have any errors in your console, please share them so I can help.

**Comment by ayushssshhh** [comment]
![image](https://github.com/adrianhajdin/project_threejs_ai/assets/109751187/899be984-edf6-48b9-9300-97b85634fa8f)

my shirt model is by default black and does not change it's color

**Comment by K4R7IK** [comment]
@ayushssshhh You can refer to my codebase, if the issue isn't fixed yet.

**Comment by Nishant-chaudhari-7** [comment]
> @ayushssshhh You can refer to my codebase, if the issue isn't fixed yet.

bro can you please help me
